packages/edit-post/src/components/header/test/index.js

```diff
@@ -36,7 +36,7 @@ describe( 'PostPublishButtonOrToggle should render a', () => {
 	it( 'toggle when post is not (1), (2), (3), the viewport is >= medium, and the publish sidebar is enabled', () => {
 		render( <PostPublishButtonOrToggle isPublishSidebarEnabled /> );
 		expect(
-			screen.getByRole( 'button', { name: 'Publish' } )
+			screen.getByRole( 'button', { name: 'Submit for Review' } )
```

Comment:
I'm not 100% sure about this change and how to manually test it. Any help is appreciated. --cc @oandregal . It's probably related to this change: https://github.com/WordPress/gutenberg/pull/60456/files#diff-4cca17f335ccfe467010f5a17979501b388e3f2c4825a00b80644baa8bda9375L182

Reply:
Looking at the [bug report](https://github.com/WordPress/gutenberg/issues/10475) that prompted me to create the PR, the issue was that the "Submit for review" string is too long in some languages — hence the use of just "Publish" in smaller viewports.

This is how to reproduce the current behavior:

- with a "contributor" role (one that doesn't have the `publish_posts` capabilities), create a post
- disable pre-publish checks (Options > Preferences modal)
- switch to a small viewport

The expected result is that the button shows "Publish" instead of "Submit for review".

https://github.com/WordPress/gutenberg/assets/583546/75e10ef0-17cc-4b28-bc74-a24fe0be82a5

Reply:
Thank you so much!